packages/flutter/lib/src/cupertino/context_menu.dart

```diff
@@ -676,11 +676,9 @@ class _DecoyChildState extends State<_DecoyChild> with TickerProviderStateMixin
 
     _boxDecoration = DecorationTween(
       begin: const BoxDecoration(
-        color: Color(0xFFFFFFFF),
         boxShadow: <BoxShadow>[],
       ),
       end: const BoxDecoration(
-        color: Color(0xFFFFFFFF),
```

Comment:
I noticed that this same kind of BoxDecoration with a white color is created in 2 of the code snippets in this file as well. Can you take a look at those too? Maybe the solution is just to remove the color there, too.

Reply:
Yes, there were similar usages of white color in the same file in 2 instances, one in the doc comments for `CupertinoContextMenuBuilder`, and `kOpenBorderRadius`, and I removed them as well, and there was no other usage of White Color.